src/actions/index.js

```diff
@@ -10,9 +10,15 @@ export const DECREMENT = 'DECREMENT';
 // is left to the reducer(s).
 
 export const increment = () => {
-  // Fill in this function
+    // Fill in this function
+    return {
+        type: INCREMENT 
+}
 };
 
 export const decrement = () => {
-  // Fill in this function
-};
+    // Fill in this function
+    return {
+        type: DECREMENT,
+    };
+};
```

Comment:
You could use an implicit return here:
```javascript
export const decrement = () => ({ type: DECREMENT });
```
Notice the parens to so that we can return an object without JS reading them as the code block of our function.